src/probability/integration.lean

```diff
@@ -118,4 +120,132 @@ lintegral_mul_eq_lintegral_mul_lintegral_of_independent_measurable_space
   (measurable_iff_comap_le.1 h_meas_f) (measurable_iff_comap_le.1 h_meas_g) h_indep_fun
   (measurable.of_comap_le le_rfl) (measurable.of_comap_le le_rfl)
 
+/-- This proves that if `f` and `g` are independent and almost everywhere measurable,
+   then `E[f * g] = E[f] * E[g]` (slightly generalizing
+   `lintegral_mul_eq_lintegral_mul_lintegral_of_indep_fun`). -/
+lemma lintegral_mul_eq_lintegral_mul_lintegral_of_indep_fun' [measurable_space α] {μ : measure α}
+  {f g : α → ℝ≥0∞} (h_meas_f : ae_measurable f μ) (h_meas_g : ae_measurable g μ)
+  (h_indep_fun : indep_fun f g μ) :
+  ∫⁻ a, (f * g) a ∂μ = ∫⁻ a, f a ∂μ * ∫⁻ a, g a ∂μ :=
+begin
+  rcases h_meas_f with ⟨f',f'_meas,f'_ae⟩,
+  rcases h_meas_g with ⟨g',g'_meas,g'_ae⟩,
+  have fg_ae : f * g =ᵐ[μ] f' * g' := f'_ae.mul g'_ae,
+  rw [lintegral_congr_ae f'_ae, lintegral_congr_ae g'_ae, lintegral_congr_ae fg_ae],
+  apply lintegral_mul_eq_lintegral_mul_lintegral_of_indep_fun f'_meas g'_meas,
+  exact h_indep_fun.ae_eq f'_ae g'_ae
+end
+
+/-- This shows that the product of two independent, integrable, real_valued random variables
+  is itself integrable. -/
+lemma indep_fun.integrable_mul [measurable_space α] {μ : measure α}
+  {β : Type*} [measurable_space β] [normed_division_ring β] [borel_space β]
```

Comment:
Can you change all measurable_space arguments to implicit arguments?

Reply:
Done.